wgengine/pendopen.go

```diff
@@ -86,6 +86,35 @@ func (e *userspaceEngine) trackOpenPreFilterIn(pp *packet.Parsed, t *tstun.Wrapp
 	return
 }
 
+var (
+	appleIPRange     = netip.MustParsePrefix("17.0.0.0/8")
+	canonicalIPRange = netip.MustParsePrefix("185.125.190.0/24")
+)
+
+// isOSNetworkProbe reports whether the target is likely a network
+// connectivity probe target from e.g. iOS or Ubuntu network-manager.
+//
+// iOS likes to probe Apple IPs on all interfaces to check for connectivity.
+// Don't start timers tracking those. They won't succeed anyway. Avoids log
+// spam like:
+func (e *userspaceEngine) isOSNetworkProbe(dst netip.AddrPort) bool {
+	// iOS had log spam like:
+	// open-conn-track: timeout opening (100.115.73.60:52501 => 17.125.252.5:443); no associated peer node
+	if runtime.GOOS == "ios" && dst.Port() == 443 && appleIPRange.Contains(dst.Addr()) {
+		if _, ok := e.PeerForIP(dst.Addr()); !ok {
+			return true
+		}
+	}
+	// NetworkManger; https://github.com/tailscale/tailscale/issues/13687
```

Comment:
Oh, and this should probably be `NetworkManager` (not `NetworkManger`)

Reply:
done